addons/interaction/CfgVehicles.hpp

```diff
@@ -1,25 +1,45 @@
+#define MACRO_PORTABLELIGHTSACTIONS(var1) \
+    class ACE_Actions { \
+        class ACE_MainActions { \
+            displayName = CSTRING(MainAction); \
+            selection = ""; \
+            distance = 2; \
+            condition = "true"; \
+            class ACE_SwitchLamp { \
+                displayName = CSTRING(var1); \
+                condition = QUOTE(alive _target); \
+                statement = QUOTE([ARR_1(_target)] call DFUNC(switchLamp)); \
```

Comment:
`ARR_N` is unnecessary when you have only 1 element, `[_target] call ...` works.

Even better would be `_target call ...` without array.